HttpRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.example.dk.mapsviabt;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import retrofit.http.GET;
+import retrofit.Callback;
+import retrofit.http.Query;
+/**
+ * Created by Karteek Dhara on 5/20/2016.
+ */
+
+//The url for all these is same currently but will be changed if necessary.
+public interface HttpRequest {
+    @GET("/api/?format=json")
+    void getMarkers(Callback<ServerData> cb);
```

Comment:
yeah I dint know what to put for one of them. I will change after confirming the path with kvn. He is not done with the work so..